src/strongbox/ui/jfx.clj

```diff
@@ -490,12 +575,37 @@
     (callback)
     nil))
 
+;; https://github.com/cljfx/cljfx/blob/babc2f09e4827efb29f859a442a1658d82169a62/examples/e25_radio_buttons.clj
+(defn radio-group
+  [{:keys [options value on-action label-coercer container-id container-type]}]
+  {:fx/type fx/ext-let-refs
+   :refs {::toggle-group {:fx/type :toggle-group}}
+   :desc {:fx/type (or container-type :h-box)
+          :id (or container-id (utils/unique-id))
+          :children (for [option options]
+                      {:fx/type :radio-button
+                       :toggle-group {:fx/type fx/ext-get-ref
+                                      :ref ::toggle-group}
+                       :selected (= option value)
+                       :text ((or label-coercer str) option)
+                       :on-action (partial on-action option)})}})
+
+(defn component-instance
+  [desc]
+  (-> desc fx/create-component fx/instance))
+
 ;;
 
 (def INSTALLED-TAB 0)
 (def SEARCH-TAB 1)
 (def LOG-TAB 2)
 
+(defn label
```

Comment:
I don't think we're using this, rm if we're not.

Otherwise, docstr, defn-spec